python/tvm/dlight/gpu/decode_gemv.py

```diff
@@ -0,0 +1,240 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=missing-docstring
+"""A fallback schedule rule for GPU operators."""
+# pylint: disable=invalid-name
+
+from typing import List, Union, Optional
+from functools import reduce
+import tvm
+from tvm import tir
+from tvm.tir import Schedule
+from tvm._ffi import get_global_func
+from tvm.target import Target
+from tvm.tir.schedule import BlockRV
+
+from ..base import BlockInfo, ScheduleRule, try_inline
+
+
+class DecodeGEMV(ScheduleRule):
+    def __init__(self) -> None:
+        super().__init__()
+        self.is_trivial_binding = get_global_func("tir.schedule.IsTrivialBinding")
```

Comment:
How about exposing such interfaces to the python side, instead `get_global_func`?